### PR TITLE
147 use specific helm chart version for individual applications using cwf

### PR DESCRIPTION
**Pull request by 12345pradeep**

### CODE REVIEW

File: jenkins-pipeline-scripts/deploy-calance-services/Jenkinsfile [hunk 1]
```diff
@@ -82,14 +82,23 @@ node("$cloudName") {
                 ]]
             ])
 
-            sh("""helm upgrade ${cleanedReleaseName} oci://ghcr.io/calance-us/calance-services \
+
+            def application_name = "${cleanedReleaseName}-${CLUSTER_ENVIRONMENT}"
```
Comment (agoyalcalance) [inline jenkins-pipeline-scripts/deploy-calance-services/Jenkinsfile:86]: why are you using `def` now? I see everything was changed to `String` keyword previously. Shouldn't we maintain consistency?

**Review by agoyalcalance — CHANGES_REQUESTED** [review]
apart from this feedback rest LGTM

**Review by ypmittal — DISMISSED** [review]
LGTM

**Review by ypmittal — APPROVED** [review]
LGTM

**Pull request merged**